pyshelf/search_parser.py

```diff
@@ -71,18 +73,23 @@ def _format_sort_criteria(self, sort_string):
         """
         sort_criteria = {}
         flag_list = []
+        sort_list = sort_string.split(",")
+        sort_criteria["field"] = sort_list.pop(0).strip()
 
-        for string in sort_string.split(","):
-            string = string.strip()
+        for sort_string in sort_list:
+            sort_string = sort_string.strip()
+            sort_type = SortType.get_alias(sort_string)
+            sort_flag = SortFlag.get_alias(sort_string)
 
-            if hasattr(SortType, string):
-                sort_criteria["sort_type"] = string
-            elif hasattr(SortFlag, string):
-                flag_list.append(string)
-            else:
-                sort_criteria["field"] = string
+            if sort_type:
+                sort_criteria["sort_type"] = sort_type
```

Comment:
This is more of a question:

With this current functionality we pick the last "sort_type" from the users perspective.  This sort of makes sense from their perspective.

If we only found it once, we probably would save (very little) processing power.  What you think?  Either way, can you add a comment explaining our decision?